go.mod

```diff
@@ -6,17 +6,17 @@ require (
 	github.com/container-storage-interface/spec v1.8.0
 	github.com/google/gofuzz v1.2.0 // indirect
 	github.com/imdario/mergo v0.3.12 // indirect
-	github.com/kubernetes-csi/csi-lib-utils v0.13.0
+	github.com/kubernetes-csi/csi-lib-utils v0.15.0
```

Comment:
Keep this at 0.14.0 until we resolve the dependency issue.

Reply:
This is causing dependency issue only for certain libraries from the otel package. It did not throw any error for "external-resizer" in `make test`. SHould we still use `v14` (may be to keep all sidecars on same dependency?) then we might want to as well revisit [node-driver-registrar](https://github.com/kubernetes-csi/node-driver-registrar/pull/323/files)

Reply:
I think it's better to be consistent across all the sidecars.

Reply:
Updated `csi-lib-utils` to `v0.14.0`